source/common/network/utility.cc

```diff
@@ -200,7 +214,12 @@ Utility::parseInternetAddressAndPortNoThrow(const std::string& ip_address, bool
   }
   sa4.sin_family = AF_INET;
   sa4.sin_port = htons(port64);
-  return Address::InstanceFactory::createInstancePtrOrDie<Address::Ipv4Instance>(&sa4);
+  StatusOr<Address::InstanceConstSharedPtr> address =
+      Address::InstanceFactory::createInstancePtrOrDie<Address::Ipv4Instance>(&sa4);
+  if (address.ok()) {
+    return *address;
+  }
```

Comment:
factor out a function method
```
  T objectOrNull(StatusOr<T>)
```
and use it 3x here

Reply:
thanks, will refactor.